exercises/grade-school/GradeSchoolTest.cs

```diff
@@ -1,61 +1,135 @@
+using GradeSchool;
```

Comment:
No it is not, it is only necessary because you defined your school in that namespace. Removing the namespace in the school implementation should fix it.